docs/en/stack/ml/anomaly-detection/ml-troubleshooting.asciidoc

```diff
@@ -188,3 +188,39 @@ POST _ml/set_upgrade_mode?enabled=false&timeout=10m
 DELETE temp_ml_config
 --------------------------------------------------
 // TEST[skip:TBD]
+
+[discrete]
+[[ml-debian8-memory]]
+== Suboptimal job assignment on Debian 8
+
+Where possible {ml} jobs are assigned to nodes based on the memory requirement
+of the job and the memory available on the node. However, in certain cases, the
+amount of memory on a node cannot be accurately determined, and then jobs are
+assigned by balancing numbers of jobs per {ml} node. It may lead to
+a situation where all the jobs with high memory requirements are on one node and the less memory-intensive jobs on
+another.
+
+{es} fails to determine the amount of memory on a machine that is
+running Debian 8 with the default Cgroups setup, and certain updates of Java
+versions earlier than Java 15.  For example, Java 8u271 is known to be affected
+while Java 8u272 is not. Java 15 was fixed from its initial release.
+
+If you are running {es} on Debian 8 with an old version of Java and
+have not already modified the Cgroups setup then it is recommended to do one of the
+following:
+
+1. Upgrade Java to version 15.
+2. Upgrade to the latest Java update for the version of Java you are running.
+3. Enable the "memory" Cgroup by editing `/etc/default/grub` and adding:
+
+```
+GRUB_CMDLINE_LINUX_DEFAULT="quiet cgroup_enable=memory swapaccount=1"
+```
+
+Then update your GRUB configuration by running:
+
+```
+sudo update-grub
+```
+
+And finally reboot the machine.
```

Comment:
Not a big deal, but I'd put this in the ordered list too:
```suggestion
4.  Reboot the machine.
```

Reply:
Although I originally committed this, on re-reading more carefully I remembered this is part of step 3.  You only need to choose any 1 from steps 1, 2 and 3.  I have tried to add `+` and `--` markings to bind the whole of step 3 together better.

But is it against the Elastic style to put "pick any one" options in numbered lists?  Should these three options be in a bulleted list instead?

Reply:
Oh wow, I totally skimmed over the "do one of the following...".  I agree they should be a bulleted list instead of a numbered list to make that more obvious.